practice-project-api/routes/calculateRoutes.ts

```diff
@@ -0,0 +1,34 @@
+import { Router, Request, Response } from "express";
+
+export const calculateRoute = Router();
+
+calculateRoute.post("/calculate", (req: Request, res: Response, next) => {
+
+  type calculatorOperation = '*' | '/' | '+' | '-';
```

Comment:
make this an enum so you can access them in the switch case in line 22 without hard coding. The reason you should do this is because if you were to add/change an operation, it's managed through the enum and not through you remembering where you hard coded values